### ISSUE TITLE
Glob style pathspecs are not handled correctly, and no warning is given

### ISSUE THREAD

**Issue by j13k** [j13k/yaml-lint#7]
## Problem

As reported in feature request https://github.com/j13k/yaml-lint/issues/3 and https://github.com/lmc-eu/php-coding-standard/pull/8, there is an issue with glob style path specs (e.g. `*.yml`) not being handled correctly—only the last file in the set of filenames is parsed, but this is not clear in the output. This is confusing for users and could result in false reports. 

## Proposed Fix

Until the tool is upgraded to handle multiple files, it should be patched to reject glob style path specs and warn the user. The documentation should also clarify this.

## Analysis

It seems `$_SERVER['argv']` expands globs automatically, behaviour which was unexpected on my part. The script loops over the values of `argv`, assigning path values to a scalar `argPath` variable, which is then parsed. When multiple paths are specified (as in case of the expanded glob file set), the value of `argPath` ultimately rests on the last filename in the list.

This scratch script illustrates the issue using the Symfony Yaml component test fixture files as an example:

```php
<?php print var_export($_SERVER['argv']) . PHP_EOL;
```

The script output demonstrates the implicit glob expansion:

```
$ php php-argv-test.php vendor/symfony/yaml/Tests/Fixtures/*.yml
array (
  0 => 'php-argv-test.php',
  1 => 'vendor/symfony/yaml/Tests/Fixtures/YtsAnchorAlias.yml',
  2 => 'vendor/symfony/yaml/Tests/Fixtures/YtsBasicTests.yml',
... lines deleted ...
  28 => 'vendor/symfony/yaml/Tests/Fixtures/sfTests.yml',
  29 => 'vendor/symfony/yaml/Tests/Fixtures/unindentedCollections.yml',
)
```

In the case of this path spec, only the last file, `vendor/symfony/yaml/Tests/Fixtures/unindentedCollections.yml`, is processed.

**Comment by j13k** [comment]
Fixed in https://github.com/j13k/yaml-lint/commit/f12fbc8c3c5ea30a2898d9827444224534b17d92.
